src/components/TableView.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import {TableColumnData, TableData, TableRowData} from "../api/Interfaces";
+import {Card, Table} from "react-bootstrap";
+import TableRowView from "./TableRowView";
+import TableColumnTogglesView from "./TableColumnTogglesView";
+import Accordion from "react-bootstrap/Accordion";
+import {BsToggles} from "react-icons/all";
+
+interface TableViewProps {
+    table: TableData
+}
+
+interface TableViewState {
+    cols: TableColumnData[],
+    toggleColumnsExpanded: boolean
+}
+
+class TableView extends React.Component<TableViewProps> {
+    state: TableViewState;
+
+    constructor(props: TableViewProps) {
+        super(props);
+
+        this.state = {
+            cols: props.table.columns,
+            toggleColumnsExpanded: false,
+        };
+    }
+
+    toggleColumn(cols: TableColumnData[]) {
+        this.setState({cols: cols});
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/ranimepiracy/index/pullRequest?prid=7272556)
```suggestion
        this.setState({cols});
```